src/Pages/joeunlee/Login/Login.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import "./Login.scss"
+import "../../../Styles/common.scss"
+import "../../../Styles/reset.scss"
+import { withRouter } from "react-router-dom";
+
+class LoginJoEun extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      idVal: "",
+      pswVal: ""
+    }
+  }
+  // goToMain = () => {
+  //   this.props.history.push('/main-joeun')
+  // };
+  consoleId = (e) => {
+    this.setState({idVal: e.target.value})
+  }
+  consolePsw = (e) => {
+    this.setState({pswVal: e.target.value})
+  }
+  consoleVal = (e) => {
+    e.preventDefault()
+    console.log('Id :', this.state.idVal,',Password :',  this.state.pswVal)
+  }
+  render() {
+    let condition = this.state.idVal.includes('@')&&this.state.pswVal.length>=5;
```

Comment:
어떤 condition인지 의미가 드러나게 해주세요. 변수 키워드, 띄워쓰기도 신경써주세요!
```suggestion
    const isUserInfoValid = this.state.idVal.includes('@') && this.state.pswVal.length >= 5;
```